charts/hono/Chart.yaml

```diff
@@ -15,7 +15,7 @@ name: hono
 description: |
   Eclipse Hono™ provides remote service interfaces for connecting large numbers of IoT devices to a back end and
   interacting with them in a uniform way regardless of the device communication protocol.
-version: 2.1.5
+version: 2.1.6
```

Comment:
@vhdirk Can you also add an entry in the `Release Notes` section of the `README.md`?

Reply:
done. Changed to v2.2.0 as per https://github.com/eclipse/packages/pull/416#discussion_r1001436531